index.html

```diff
@@ -3852,9 +3852,14 @@ <h2 id="AudioParam">
             <p>
               Cancels all scheduled parameter changes with times greater than
               or equal to <code>cancelTime</code>. Active
-              <code><a>setTargetAtTime</a></code> automations (those with
-              <code>cancelTime</code> less than the supplied time value) will
-              also be cancelled. Any hold values scheduled by <a href=
+              <code><a>linearRampToValueAtTime</a></code>,
+              <code><a>exponentialRampToValueAtTime</a></code>,
+              <code><a>setTargetAtTime</a></code> and
+              <code><a>setValueCurveAtTime</a></code> automations (those with
+              <code>cancelTime</code> less than the automation's end time) will
```

Comment:
I would simplify this to say something like "any automation scheduled to occur after `cancelTime`, even if the automation is currently running is removed.  This may cause discontinuities,..."